docs/package-yanking-policies.md

```diff
@@ -0,0 +1,149 @@
+# Crafting a package deletion policy
+
+Last updated: April 2025
```

Comment:
Minor: if you'd like this work to be attributed to you, you can add an author line like we have on https://github.com/ossf/wg-securing-software-repos/blob/main/docs/trusted-publishers-for-all-package-repositories.md?plain=1#L3 or https://github.com/ossf/wg-securing-software-repos/blob/main/docs/principles-for-package-repository-security.md?plain=1#L3